js/libs/keycloak-js/src/keycloak.js

```diff
@@ -176,6 +176,9 @@ function Keycloak (config) {
                     options.prompt = 'none';
                 }
 
+                if (initOptions && initOptions.locale) {
+                    options.locale = initOptions.locale
```

Comment:
I don't see this option being handled in `createLoginUrl`, so I don't believe this parameter is actually being added (unless I missed something). Did you test this code works?

Reply:
```
if (options && options.locale) {
    url += '&ui_locales=' + encodeURIComponent(options.locale);
}
```
In createLoginUrl，I think the option locale has been handled in above code. Did i understand it wrong?
I tested this code in previous version.

Reply:
Ahhh, yes. You are completely right, consider this resolved :+1: